tests/Contracts/DocumentsQueryTest.php

```diff
@@ -36,4 +36,41 @@ public function testSetOffset(): void
 
         self::assertSame(['offset' => 5], $data->toArray());
     }
+
+    public function testSetIdsWithStringArray(): void
+    {
+        $data = (new DocumentsQuery())->setIds(['1', '2', '3']);
+
+        self::assertSame(
+            implode(',', ['1', '2', '3']),
+            $data->toArray()['ids']
+        );
+    }
+
+    public function testSetIdsWithIntegerArray(): void
+    {
+        $data = (new DocumentsQuery())->setIds([1, 2, 3]);
```

Comment:
as you test same thing in multiple methods, you could write a data provider and use single method for 'ids' test

Reply:
Indeed, that's much cleaner  ✨ thanks!